src/tiledimage.js

```diff
@@ -1018,7 +1018,7 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
         );
         var currentZeroRatio = this.viewport.deltaPixelsFromPointsNoRotate(
             this.source.getPixelRatio(0), true).x *
-            this._scaleSpring.current.value;
+            this._scaleSpring.current.value * $.pixelDensityRatio;
```

Comment:
Can you explain why this addition is needed? Likewise with the other change in this file. I can see how it might be good, but it's interesting that we didn't already have it here. I'm curious what you found that led you to it.

Reply:
`this.source.getPixelRatio` also need to take the pixel density into concern. I have refactored the changes in this file and moved them to `tilesource.getPixelRatio´. I think that is the correct place to do the calculations.